README.md

```diff
@@ -102,55 +167,30 @@ make install-develop
 Make sure to use them regularly while developing by running the commands `make lint` and
 `make test`.
 
-## What is a synthesizer function?
+## Compile C++ dependencies
```

Comment:
We should check PrivBayes' license.

Reply:
Can you please do that? All I did was adding the compile target to the Makefile, but the privbayes folder was already there since the beginning, so I do not know where you got it from.